.eslintrc

```diff
@@ -35,6 +35,18 @@
     "es6": true
   },
 
+  "globals": {
```

Comment:
@AlanFoster currently we don't enforce linting. Would be nice to get a gulp task for it though.